processor/src/advice/providers.rs

```diff
@@ -106,6 +109,22 @@ where
         Ok(())
     }
 
+    fn get_signature(
+        &self,
+        signature: SignatureKind,
+        pub_key: Word,
+        msg: Word,
+    ) -> Result<Vec<Felt>, ExecutionError> {
```

Comment:
Similar nit re renaming `signature` param to `kind`, here and in two other functions in this file.